xml/cl.xml

```diff
@@ -1727,7 +1727,7 @@ server's OpenCL/api-docs repository.
         <enum value="0x1296"             name="CL_COMMAND_BUFFER_REFERENCE_COUNT_KHR"/>
         <enum value="0x1297"             name="CL_COMMAND_BUFFER_STATE_KHR"/>
         <enum value="0x1298"             name="CL_COMMAND_BUFFER_PROPERTIES_ARRAY_KHR"/>
-            <unused start="0x1299" comment="Reserved for MR199"/>
+        <enum value="0x1299"             name="CL_DEVICE_COMMAND_BUFFER_SUPPORTED_QUEUE_PROPERTIES_KHR"/>
```

Comment:
Note that PR https://github.com/KhronosGroup/OpenCL-Docs/pull/899 is now also trying to claim this enum